src/style.css

```diff
@@ -0,0 +1,98 @@
+@import "./block_styles/card.css";
+@import "./block_styles/stars.css";
+
+* {
+  margin: 0;
+}
```

Comment:
Don't reset default margins using * selector (not every tag have default margins)